<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClientWindow</class>
 <widget class="QMainWindow" name="AddClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить пациента</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#48FFD5</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Адрес</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="textFIO"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дата рождения</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Паспорт</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="textPhone"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="textPassport"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="textAddress"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="textEmail"/>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateBirth">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>1997</year>
         <month>1</month>
         <day>8</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
